packages/kit/test/apps/basics/src/routes/routing/_tests.js

```diff
@@ -238,4 +241,27 @@ export default function (test) {
 			assert.equal(await page.url(), 'https://www.google.com/');
 		}
 	);
+
+	test('watch new route in dev', '/routing', async ({ page, base, js }) => {
+		if (!is_dev || js) {
+			return;
+		}
+
+		// hash the filename so that it won't conflict with
+		// future test file that it has the same name
+		const route = 'bar' + new Date().valueOf();
+		const content = 'Hello new route';
+		const __dirname = path.dirname(fileURLToPath(import.meta.url));
+		const filePath = path.join(__dirname, `${route}.svelte`);
+
+		try {
+			fs.writeFileSync(filePath, `<h1>${content}</h1>`);
+			await Promise.resolve();
```

Comment:
Don't know if there is any way to wait for the manifest update. Maybe I should just wait for a few seconds. It might also let tests pass in js. I currently disable the test in js because it has to wait for the dev server to send updates to the client.